sql/sql_class.cc

```diff
@@ -3258,25 +3261,45 @@ static File create_file(THD *thd, char *path, sql_exchange *exchange,
     return -1;
   }
 
-  if (!access(path, F_OK))
+  file_exists_already= my_stat(path, &file_stat, MYF(0));
+
+/* Probably a fifo could be used under Windows, but we're not
```

Comment:
```suggestion
/*
  Probably a fifo could be used under Windows, but we're not
  attempting it at the moment.
*/
```
Its odd, but consistent with existing comment format.